### ISSUE TITLE
install using yarn failed

### ISSUE THREAD

**Issue by wawansimanggalung** [toverux/ngx-sweetalert2#34]
I use jhipster angular 4 generator and add sweetalert angular dependency

`yarn add sweetalert2 @toverux/ngx-sweetalert2`

And I got this error on running using webpack dev server

```
Uncaught Error: Unexpected value '[object Object]' imported by the module 'MyModule'. Please add a @NgModule annotation.
    at syntaxError (webpack-internal:///./node_modules/@angular/compiler/@angular/compiler.es5.js:1918)
    at eval (webpack-internal:///./node_modules/@angular/compiler/@angular/compiler.es5.js:15614)
    at Array.forEach (<anonymous>)
    at CompileMetadataResolver.getNgModuleMetadata (webpack-internal:///./node_modules/@angular/compiler/@angular/compiler.es5.js:15597)
    at CompileMetadataResolver.getNgModuleSummary (webpack-internal:///./node_modules/@angular/compiler/@angular/compiler.es5.js:15539)
    at eval (webpack-internal:///./node_modules/@angular/compiler/@angular/compiler.es5.js:15612)
    at Array.forEach (<anonymous>)
    at CompileMetadataResolver.getNgModuleMetadata (webpack-internal:///./node_modules/@angular/compiler/@angular/compiler.es5.js:15597)
    at CompileMetadataResolver.getNgModuleSummary (webpack-internal:///./node_modules/@angular/compiler/@angular/compiler.es5.js:15539)
    at eval (webpack-internal:///./node_modules/@angular/compiler/@angular/compiler.es5.js:15612)
```

Thanks

**Comment by toverux** [comment]
Thanks for reporting. I'll try to take a look today.
Also, maybe I have an idea, could you attach your yarn.lock?

**Comment by wawansimanggalung** [comment]
Okay, I rename it to yarn.log :D
[yarn.log](https://github.com/toverux/ngx-sweetalert2/files/1487436/yarn.log)

Thanks :D

**Comment by toverux** [comment]
Thanks, I understand what the problem is. The issue is on my side indeed, expect a fix in the next few hours.
In the meantime, and if I'm right, I think that you _may_ be able to workaround it by deleting node_modules and yarn.lock and re-run `yarn install`. That should dedupe the Angular dependencies, which is what causes the problem here.

Sorry for the inconvenience!

**Comment by jpduranhe** [comment]
hi , 

I have angular 5, when install following you instructions show me this error message

ERROR in node_modules/sweetalert2/sweetalert2.d.ts(209,17): error TS2300: Duplicate identifier 'SweetAlertType'.
node_modules/sweetalert2/sweetalert2.d.ts(211,17): error TS2300: Duplicate identifier 'SweetAlertInputType'.
node_modules/sweetalert2/sweetalert2.d.ts(217,22): error TS2300: Duplicate identifier 'SweetAlertInputOptions'.
node_modules/sweetalert2/sweetalert2.d.ts(221,22): error TS2300: Duplicate identifier 'SweetAlertInputAttributes'.
node_modules/sweetalert2/sweetalert2.d.ts(678,5): error TS2300: Duplicate identifier 'default'.
src/assets/vendors/bower_components/sweetalert2/sweetalert2.d.ts(198,17): error TS2300: Duplicate identifier 'SweetAlertType'.

src/assets/vendors/bower_components/sweetalert2/sweetalert2.d.ts(200,17): error TS2300: Duplicate identifier 'SweetAlertInputType'.
src/assets/vendors/bower_components/sweetalert2/sweetalert2.d.ts(202,17): error TS2300: Duplicate identifier 'SweetAlertInputOptions'.
src/assets/vendors/bower_components/sweetalert2/sweetalert2.d.ts(204,17): error TS2300: Duplicate identifier 'SweetAlertInputAttributes'.
src/assets/vendors/bower_components/sweetalert2/sweetalert2.d.ts(232,9): error TS2403: Subsequent variable declarations must have the same type.  Variable 'html' must be of type 'string | JQ
uery', but here has type 'string'.
src/assets/vendors/bower_components/sweetalert2/sweetalert2.d.ts(402,9): error TS2403: Subsequent variable declarations must have the same type.  Variable 'preConfirm' must be of type '(inpu
tValue: any) => any', but here has type '(inputValue: any) => Promise<any>'.
src/assets/vendors/bower_components/sweetalert2/sweetalert2.d.ts(481,9): error TS2403: Subsequent variable declarations must have the same type.  Variable 'inputValidator' must be of type '(
inputValue: any) => string | Promise<string>', but here has type '(result: any) => Promise<void>'.
src/assets/vendors/bower_components/sweetalert2/sweetalert2.d.ts(493,9): error TS2403: Subsequent variable declarations must have the same type.  Variable 'progressSteps' must be of type 'st
ring[]', but here has type 'SweetAlertOptions[]'.
src/assets/vendors/bower_components/sweetalert2/sweetalert2.d.ts(520,5): error TS2300: Duplicate identifier 'default'.

thanks

**Comment by toverux** [comment]
There we are, it should be fixed with [v3.0.1](https://github.com/toverux/ngx-sweetalert2/releases/tag/v3.0.1) - `yarn upgrade @toverux/ngx-sweetalert2`. If not, please ensure that yarn has deduped Angular dependencies correctly (basically remove node_modules and yarn.lock then regenerate them).

**Comment by wawansimanggalung** [comment]
Works like a charm! Thanks a lot! :D
